src/components/About/About.jsx

```diff
@@ -36,11 +36,10 @@ function About() {
           <Col md="8">
             <Section title="About Us">
               <p className="text-center">
-                The Terasology project was and is becoming a stable platform for
-                various types of gameplay settings in a voxel world. The
-                creators and maintainers are a diverse mix of software
```

Comment:
I always liked that sentence - can we weave it in somehow?

Reply:
same here, that's why it's still in but simply in another place where it makes more sense (see below in the "Community" section)